app/views/shared/_checkbox_group.html.haml

```diff
@@ -10,4 +10,5 @@
     value_method,
     text_method,
     hint_method = hint_method,
-    small: small
+    small: small,
+    classes: classes
```

Comment:
What about if we do `classes: classes || ''` here instead?

Reply:
`classes: (defined? classes) ? classes : ''`